### PR TITLE
add scaling function parameter

### PR DESCRIPTION
**Pull request by yffbit**

### CODE REVIEW

**Comment by ahainaut** [comment]
Hi,
Thank you for your contribution !
Could you please elaborate on the effect of this parameter and the necessity of adding it to the configuration? I know that it has an influence on the Lipschitz constant of the scaling function but I don’t think adding this as a new hyperparameter will help for optimisation. For example, in MuZero paper, this parameter is fixed to 0.001 for all experiments.

**Comment by ahainaut** [comment]
Hi,
Thank you for your contribution !
Could you please elaborate on the effect of this parameter and the necessity of adding it to the configuration? I know that it has an influence on the Lipschitz constant of the scaling function but I don’t think adding this as a new hyperparameter will help for optimisation. For example, in MuZero paper, this parameter is fixed to 0.001 for all experiments.

**Comment by yffbit** [comment]
Adding epsilon parameter in `support_to_scalar` and `scalar_to_support` is just for test. In the paper, https://arxiv.org/abs/1805.11593, the default value is 0.01. MuZero changes it to 0.001. May be better not to write it as a constant in the formula.

File: replay_buffer.py [hunk 1]
```diff
@@ -197,30 +197,29 @@ def update_priorities(self, priorities, index_info):
...
         else:
             value = 0
-
-        for i, reward in enumerate(
-            game_history.reward_history[index + 1 : bootstrap_index + 1]
-        ):
-            value += (
-                reward
-                if game_history.to_play_history[index]
-                == game_history.to_play_history[index + 1 + i]
```
Comment (ydy93) [inline replay_buffer.py:221]: I think it should be `game_history.to_play_history[index] == game_history.to_play_history[index + i]`

**Comment by ahainaut** [comment]
Could you please run some tests to see the influence of the epsilon parameter on the training so we can know more precisely  if epsilon could be a key parameter for optimisation ?

**Pull request closed without merging**